.azure/pipelines/azure-pipelines-mirror-within-azdo.yml

```diff
@@ -4,58 +4,66 @@ trigger:
   branches:
     include:
     - internal/release/6.0
+    
+parameters:
+# Run the pipeline manually (usually disallowed)
+- name: manualRun
+  default: false
+  displayName: Are you sure you want to run this pipeline manually?
```

Comment:
Could you explain the point of this confirmation❔

Reply:
The use case of this pipeline is pretty narrow, and given that we're making users explicitly opt-in to running it manually, I think it makes sense to ask for confirmation.